tests/test_streams.rs

```diff
@@ -277,12 +288,18 @@ fn test_assorted_2() {
 
     // call xpending here...
     // this has a different reply from what the count variations return
-    let reply: StreamPendingReply = con.xpending("k99", "g99").unwrap();
-    assert_eq!(reply.count, 3);
-    assert_eq!(reply.start_id, "1000-1");
-    assert_eq!(reply.end_id, "1001-1");
-    assert_eq!(reply.consumers.len(), 1);
-    assert_eq!(reply.consumers[0].name, "c99");
+    let data_reply: StreamPendingReply = con.xpending("k99", "g99").unwrap();
+
```

Comment:
I think we might be able to add this here: `assert_ne!(data_reply, StreamPendingReply::Empty)`